features/post-term.feature

```diff
@@ -199,3 +199,18 @@ Feature: Manage post term
     Then STDOUT should be a table containing rows:
       | term_id | name | slug |
       | 3       | 4    | 4    |
+
+  Scenario: Remove all terms from post
+    Given a WP install
+
+    When I run `wp post term remove 1 category --all`
+    Then STDOUT should be:
+      """
+      Success: Removed term.
+      """
+
```

Comment:
We should also have a test that the default category was added back to the post.